cert/cert.go

```diff
@@ -119,7 +123,13 @@ func NewSignedCert(cfg Config, key crypto.Signer, caCert *x509.Certificate, caKe
 	if err != nil {
 		return nil, err
 	}
-	return x509.ParseCertificate(certDERBytes)
+
+	parsedCert, err := x509.ParseCertificate(certDERBytes)
+	if err == nil {
+		logrus.Infof("certificate %v signed by %v: notBefore=%v notAfter=%v",
```

Comment:
Is this for debugging primarily or is it something that we would expect user/operators to dig into their logs looking for?

Reply:
We already log the expiration at info level, so I thought it would be nice to also log the issuance so that anyone looking at the logs knows that the expiration has been extended. Do you think this (and the expiration) should be debug?

Reply:
It's hard to say. With logs I try to err on the side of minimal logs in "prod" aka the default which tends to route stuff to debug. I think with this change you have a number of infos that should probably be debugs however it would be valuable and informative to print this to info once per event.

Reply:
Our embedded certificate stuff feels like a bit of a black box at the moment. These logs should only trigger when the cluster is first created, and then again 9 months later when the certs are a about to expire, so these should not be very chatty despite being at info level.